doc-content/enterprise-only/vehicle-route-planning/run-vrp-manually-proc.adoc

```diff
@@ -0,0 +1,76 @@
+[id='run-vrp-manually-proc_{context}']
+= Configure and run OptaWeb Vehicle Routing manually
+
+The easiest way to run OptaWeb Vehicle Routing is to use the `runlocally.sh` script. However, if Bash is not available on your system you can manually complete the steps that the `runlocally.sh` script performs.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+.Procedure
+. Download routing data.
++
+The routing engine requires geographical data to calculate the time it takes vehicles to travel between locations.
+You must download and store OpenStreetMap (OSM) data files on the local file system before you run OptaWeb Vehicle Routing.
++
+NOTE: The OSM data files are typically between 100 MB to 1 GB and take time to download so it is a good idea to download the files before building or starting the OptaWeb Vehicle Routing application.
+
+.. Open `http://download.geofabrik.de/` in a web browser.
+.. Click a region in the *Sub Region* list, for example *Europe*. The sub region's page opens.
+.. In the *Sub Regions* table, download the OSM file (`.osm.pbf`) for a country, for example Belgium.
+
+. Create the data directory structure.
++
+OptaWeb Vehicle Routing reads and writes several types of data on the file system.
+It reads OSM (OpenStreetMap) files from the `openstreetmap` directory, writes a road network graph to the `graphhopper` directory, and persists user data in a directory called `db`.
+Create a new directory dedicated to storing all of these data to make it easier to upgrade to a newer version of OptaWeb Vehicle Routing in the future and continue working with the data you created previously.
+
+.. Create the `openstreetmap` directory in your user account `home` directory, for example:
```

Comment:
I guess there is one step missed - first you need to create a data dir with data-dir-name

Reply:
That's implied.

Reply:
Defect is that in enterprise we got this
![image](https://user-images.githubusercontent.com/21653197/87543755-89ade580-c6a5-11ea-99d8-2f131b320620.png)
and nothing mentioned about the data-dir-name and that leads to failure in manual installation because that makes it impossible for the user to figure out what data-dir-name should be. At the same time, we got a strict rule that it should be ".optaweb-vehicle-routing" and nothing else

So the problem is that this part has been copied from the community. In community I found variable definition for {data-dir-name} = 
see http://github.com/dupliaka/optaweb-vehicle-routing/blob/master/optaweb-vehicle-routing-docs/src/main/asciidoc/attributes.adoc#L1-L1

1) But by some reason that was not substituted in this enterprise version (please check that you have committed attributes adoc)
2) Can we add a first step on that user need to create ".optaweb-vehicle-routing"  first bcs that is needed to be done manually to put there OpenStreetMap folder

Reply:
Updated

Reply:
1)This step has to be changed:
.. Create the `openstreetmap` directory in your user account `home` directory, for example: >>
Create the `openstreetmap` directory in the {VRP-DATA-DIR} directory, for example:

Reply:
Updated